src/color/p5.Color.js

```diff
@@ -403,7 +403,7 @@ var colorPatterns = {
  * For a number of different inputs, returns a color formatted as [r, g, b, a]
  * arrays, with each component normalized between 0 and 1.
  *
- * @param {Array-like} args An 'array-like' object that represents a list of
+ * @param {Array|Object} args An 'array-like' object that represents a list of
```

Comment:
oh I see further down it says Number[] for some other color methods, maybe we go with that then?